### ISSUE TITLE
Feature request: AMT commands in the Group Actions window

### ISSUE THREAD

**Issue by mrflaunted** [Ylianst/MeshCentral#2635]
One of those "nice to have" kind of features. If we have building powerdowns and graceful shutdowns don't work our fall back is to shut down via AMT. Would be nice for support to have the ability to check off a few hosts and send the powerdown\reset\powerup command via the MeshCentral GUI.

Right now we can go into each one individually or use the PowerShell module.

Thoughts?

**Comment by Ylianst** [comment]
You are right that I have basic Intel AMT power operations in each device's action button, but not in the group action button.

![image](https://user-images.githubusercontent.com/1319013/118160607-5d511b80-b3d3-11eb-83dd-11b3d51f77c1.png)

**Comment by mrflaunted** [comment]
#friendlybump

**Comment by mrflaunted** [comment]
#friendlybump for some AMT love

**Comment by mrflaunted** [comment]
friendly bump @Ylianst :)

**Comment by mrflaunted** [comment]
Following up, extremely helpful for building power down\ups.

@Ylianst @Bobberty @lheckemann @kylelt
